### ISSUE TITLE
Broken armv7 image

### ISSUE THREAD

**Issue by titouan-joseph** [AlexSciFier/neonlink#72]
Hello,

It"s seems the armv7 image is broken, sharp module isn't installed

```
neolink_neonlink.1.v133t10huy6e@Hulk    | Config plugin initialization completed.
neolink_neonlink.1.v133t10huy6e@Hulk    | Error: 
neolink_neonlink.1.v133t10huy6e@Hulk    | Something went wrong installing the "sharp" module
neolink_neonlink.1.v133t10huy6e@Hulk    | 
neolink_neonlink.1.v133t10huy6e@Hulk    | Cannot find module '../build/Release/sharp-linuxmusl-armv7.node'
neolink_neonlink.1.v133t10huy6e@Hulk    | Require stack:
neolink_neonlink.1.v133t10huy6e@Hulk    | - /app/node_modules/sharp/lib/sharp.js
neolink_neonlink.1.v133t10huy6e@Hulk    | - /app/node_modules/sharp/lib/constructor.js
neolink_neonlink.1.v133t10huy6e@Hulk    | - /app/node_modules/sharp/lib/index.js
neolink_neonlink.1.v133t10huy6e@Hulk    | 
neolink_neonlink.1.v133t10huy6e@Hulk    | Possible solutions:
neolink_neonlink.1.v133t10huy6e@Hulk    | - Install with verbose logging and look for errors: "npm install --ignore-scripts=false --foreground-scripts --verbose sharp"
neolink_neonlink.1.v133t10huy6e@Hulk    | - Install for the current linuxmusl-armv7 runtime: "npm install --platform=linuxmusl --arch=armv7 sharp"
neolink_neonlink.1.v133t10huy6e@Hulk    | - Consult the installation documentation: https://sharp.pixelplumbing.com/install
neolink_neonlink.1.v133t10huy6e@Hulk    |     at Object.<anonymous> (/app/node_modules/sharp/lib/sharp.js:37:9)
neolink_neonlink.1.v133t10huy6e@Hulk    |     at Module._compile (node:internal/modules/cjs/loader:1233:14)
neolink_neonlink.1.v133t10huy6e@Hulk    |     at Module._extensions..js (node:internal/modules/cjs/loader:1287:10)
neolink_neonlink.1.v133t10huy6e@Hulk    |     at Module.load (node:internal/modules/cjs/loader:1091:32)
neolink_neonlink.1.v133t10huy6e@Hulk    |     at Module._load (node:internal/modules/cjs/loader:938:12)
neolink_neonlink.1.v133t10huy6e@Hulk    |     at Module.require (node:internal/modules/cjs/loader:1115:19)
neolink_neonlink.1.v133t10huy6e@Hulk    |     at require (node:internal/modules/helpers:130:18)
neolink_neonlink.1.v133t10huy6e@Hulk    |     at Object.<anonymous> (/app/node_modules/sharp/lib/constructor.js:11:1)
neolink_neonlink.1.v133t10huy6e@Hulk    |     at Module._compile (node:internal/modules/cjs/loader:1233:14)
neolink_neonlink.1.v133t10huy6e@Hulk    |     at Module._extensions..js (node:internal/modules/cjs/loader:1287:10)
```

**Comment by titouan-joseph** [comment]
Same thing for arm64 image

**Comment by titouan-joseph** [comment]
Same thing for arm64 image

**Comment by titouan-joseph** [comment]
I try to build the image in RPI and no problem.
I try to build the image on my computer with buildx and the problem is the same.

I think is a buildx issue

**Comment by AlexSciFier** [comment]
Hello. Problem with one of the latest updates #59 when I added the Sharp library as a dependency. This library does not have pre-built binaries for Armv7 and Alpine. I can change the base image from Alpine to Ubuntu, but then the image will be almost 1 GB in size. But I think I'll just remove this feature until I find a better library for this job.

**Comment by titouan-joseph** [comment]
Maybe with the slim version you can have a smaller image

**Comment by AlexSciFier** [comment]
Fixed in 1.4.9

**Comment by titouan-joseph** [comment]
Thanks, it's working !
